bin/kawaii

```diff
@@ -49,7 +49,6 @@ if (! defined('COMPOSER_INSTALLED')) {
 
 require COMPOSER_INSTALLED;
 
-// TODO: get this parser by a file
```

Comment:
The `TODO` is in place because there is a need for it to be fixed, not for it to be removed